docs/site/examples-knative.md

```diff
@@ -95,7 +95,15 @@ examples:
     links:
     - language: powershell
       url: "/tree/master/examples/knative/powershell/kn-ps-horizon-slack-function"
-
+  - title: VMware Cloud Notification
+    usecases:
+    - item: integration
+    - item: notification
+    id: kn-ps-ngw-slack-function
+    description: Function to send a Slack notification triggered by a VMware Cloud Event.
```

Comment:
I was briefly confused about "Cloud Event". Would "Cloud Gateway Event" be more accurate (and less confusing wrt CloudEvent vs Cloud Event)?